### ISSUE TITLE
Feature Request: Tag Picker

### ISSUE THREAD

**Issue by sureshjoshi** [aidewoode/office-ui-fabric-vue#19]
Not sure if there is interest for this - but I'm looking into porting the Office UI Fabric repo's tag picker (currently using vue-select/vue-multiselect).

https://developer.microsoft.com/en-us/fabric#/components/pickers

I'm not sure if you want to keep this repo in lockstep with the baseline office-fabric-js - or if you're interested in a PR (someday) for the Fabric components

**Comment by aidewoode** [comment]
Yeah, I plan to rewrite the whole repo just use the office-ui-fabric-core like office-ui-fabric-react. 

You know it's lots of works. I don't have so such time for it, the progress will be slow, I plan to rewrite the component one by one base on office-ui-fabric-react. If you have interest, PR are always welcome.

**Comment by sureshjoshi** [comment]
I'm also investigating Vue binding for React:

https://github.com/alkin/vue-react

https://github.com/akxcv/vuera

No success so far, but it might be more maintainable since the react office repo is maintained well and documented (at the expense of maybe a larger codebase? Maybe?)

**Comment by aidewoode** [comment]
I think wrap react component to vue component maybe not a good idea. Because of the performance issues.

**Comment by sureshjoshi** [comment]
I agree that it's not a good long term strategy because of total library size, and maybe performance issues (I assume there will be some hit, but there aren't any metrics anywhere).

It would be a good middle-ground workaround while porting in components though - provides the Vue API to line up with the React one, and then progressively carve out the internals to be Vue only.. Or maybe stick it in a contrib repo? 

I still need to get something to compile correctly with Typescript first, then we'll see   :)
